test/fakes.ts

```diff
@@ -33,6 +33,7 @@ export interface FakeShellSettings {
 
 export interface FakeKubectlSettings {
     asLines? : (cmd: string) => string[] | ShellResult;
```

Comment:
asLines signature has been changed.

Reply:
This is for unit test. Create mocked kubectl instance.